Build from source without specified profile encountered errors

Build from source encountered errors with `mvn clean install -DskipTests`:

If we added profile `-Phadoop-3.1` or `-Phadoop-2.7`. Building is successful. I thought this error is related to `alluxio`. 

Can we fix it?


```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:compile (default-compile) on project smart-alluxio: Compilation failure: Compilation failure:
[ERROR] /Users/qiyuangong/Develop/Gitremote/SSM/smart-alluxio-support/smart-alluxio/src/main/java/org/smartdata/alluxio/metric/fetcher/AlluxioNamespaceFetcher.java:[158,27] constructor FileInfo in class org.smartdata.model.FileInfo cannot be applied to given types;
[ERROR]   required: java.lang.String,long,long,boolean,short,long,long,long,short,java.lang.String,java.lang.String,byte,byte
[ERROR]   found: java.lang.String,long,long,boolean,short,long,long,long,short,java.lang.String,java.lang.String,byte
[ERROR]   reason: actual and formal argument lists differ in length
[ERROR] /Users/qiyuangong/Develop/Gitremote/SSM/smart-alluxio-support/smart-alluxio/src/main/java/org/smartdata/alluxio/AlluxioUtil.java:[51,25] constructor FileInfo in class org.smartdata.model.FileInfo cannot be applied to given types;
[ERROR]   required: java.lang.String,long,long,boolean,short,long,long,long,short,java.lang.String,java.lang.String,byte,byte
[ERROR]   found: java.lang.String,long,long,boolean,short,long,long,long,short,java.lang.String,java.lang.String,byte
[ERROR]   reason: actual and formal argument lists differ in length
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :smart-alluxio
```

--- a/smart-alluxio-support/smart-alluxio/src/main/java/org/smartdata/alluxio/AlluxioUtil.java
+++ b/smart-alluxio-support/smart-alluxio/src/main/java/org/smartdata/alluxio/AlluxioUtil.java
@@ -60,7 +60,9 @@ public class AlluxioUtil {
         (short) status.getMode(),
         status.getOwner(),
         status.getGroup(),
-        (byte) 0);
+        (byte) 0,
+        (byte) 0
+    );
     return fileInfo;
   }
  

--- a/smart-alluxio-support/smart-alluxio/src/main/java/org/smartdata/alluxio/metric/fetcher/AlluxioNamespaceFetcher.java
+++ b/smart-alluxio-support/smart-alluxio/src/main/java/org/smartdata/alluxio/metric/fetcher/AlluxioNamespaceFetcher.java
@@ -167,6 +167,7 @@ public class AlluxioNamespaceFetcher {
           (short) status.getMode(),
           status.getOwner(),
           status.getGroup(),
+          (byte) 0,
           (byte) 0);
       return fileInfo;
     }

